tsconfig.json

```diff
@@ -1,19 +1,13 @@
 {
   "compilerOptions": {
-    "declaration": true,
     "target": "es5",
-    "downlevelIteration": true,
-    "lib": ["dom", "es7", "es2017.object"],
-    "outDir": "./dist/",
-    "sourceMap": true,
-    "noImplicitAny": true,
-    "strictNullChecks": true,
+    "sourceMap": false,
+    "declaration": true,
+    "outDir": "./lib/",
```

Comment:
This might be an useful read overall:
>`build` - contains the compiled code of our project if applicable. Sometimes called dist
>https://pathof.dev/blog/npm-package-best-practices